src/addons/__tests__/ReactComponentWithPureRenderMixin-test.js

```diff
@@ -50,20 +50,20 @@ describe('ReactComponentWithPureRenderMixin', function() {
       getInitialState: function() {
         return {
           cut: false,
-          slices: 1,
+          slices: 1
```

Comment:
Okay. I've changed my mind here. I'll fix the dangling commas after this PR lands. I hacked together a fancy codemod tool that hooks into eslint's node API to fix the code. (which is awesome because we get zero false positives, and it obeys eslintignore/eslintrc files)